docs/content/en/share_your_findings/jira_guide.md

```diff
@@ -0,0 +1,650 @@
+
+![image](images/jira-instance-beta.png)
+
+3. Select a **Configuration Name** for this Jira Instance to use in DefectDojo. This name is simply a label for the Instance connection in DefectDojo, and does not need to be related to any Jira data.
+
+4. Select the URL for your company’s Jira instance \- likely similar to `https://**yourcompany**.atlassian.net` if you’re using a Jira Cloud installation.
+
+5. Enter an appropriate authetication method in the Username / Password fields for Jira:
+    * For standard **username / password Jira authentication**, enter a Jira Username and corresponding Password in these fields.
+    * For authentication with a **user's API token (Jira Cloud)** enter the Username with the corresponding **API token** in the password field.
+    * For authentication with a Jira **Personal Access Token (aka PAT, used in Jira Data Center and Jira Server only)**, enter the PAT in the password field.  Username is not used for authentication with a Jira PAT, but the field is still required in this form, so you can use a placeholder value here to identify your PAT.
+
+Note that the user associated with this connection have permission to create Issues and access data in your Jira instance.
+
+6. You will need to provide values for an Epic Name ID, Re-open Transition ID and Close Transition ID.  These values can be changed later.  While logged into Jira, you can access these values from the following URLs:
+- **Epic Name ID**: visit `https://\<YOUR JIRA URL\>/rest/api/2/field` and search for Epic Name. Copy the number out of cf\[number] and paste it here.
+- **Re-open Transition ID**: visit `https://\<YOUR JIRA URL\>/rest/api/latest/issue/\<ANY VALID ISSUE KEY\>/transitions? expand\-transitions.fields` to find the ID for your Jira instance. Paste it in the Reopen Transition ID field.
+- **Close Transition ID**: Visit `https://\<YOUR JIRA URL\>/rest/api/latest/issue/\<ANY VALID ISSUE KEY\>/transitions? expand\-transitions.fields` to find the ID for your Jira instance. Paste it in the Close Transition ID field.
+
+7. Select the Default issue type which you want to create Issues as in Jira. The options for this are **Bug, Task, Story** and **Epic** (which are standard Jira issue types) as well as **Spike** and **Security**, which are custom issue types. If you have a different Issue Type which you want to use, please contact [support@defectdojo.com](mailto:support@defectdojo.com) for assistance.
+
+8. Select your Issue Template, which will determine the Issue Description when Issues are created in Jira.
+
+The two types are:
+\- **Jira\_full**, which will include all Finding information in Jira Issues
+\- **Jira\_limited**, which will include a smaller amount of Finding information and metadata.
+​
+If you leave this field blank, it will default to **Jira\_full.**  If you need a different kind of template, Pro users can reach out to support@defectdojo.com
+
```

Comment:
```suggestion
The two types are:
- **Jira\_full**, which will include all Finding information in Jira Issues
- **Jira\_limited**, which will include a smaller amount of Finding information and metadata.

If you leave this field blank, it will default to **Jira\_full.**  If you need a different kind of template, Pro users can reach out to [support@defectdojo.com](mailto:support@defectdojo.com)

```